examples/tui/mouse.js

```
@@ -0,0 +1,122 @@
+  }
+
+  const padded = lines.map(line => {
+    if (line.length >= width) return line.slice(0, width);
+    return line + ' '.repeat(width - line.length);
+  });
+
+  write(codes.home + codes.clear + padded.join('\n') + codes.reset);
+}
+
+function parseMouseEvent(seq) {
+  const match = seq.match(/^\x1b\[<([0-9]+);([0-9]+);([0-9]+)([mM])$/);
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)